test/fuzz_group.cpp

```diff
@@ -207,7 +207,7 @@ Mixed construct_mixed(State& s, util::Optional<std::ostream&> log, std::string&
         }
         case 5: {
             size_t rand_char = get_next(s);
-            size_t blob_size = get_int64(s) % ArrayBlob::max_binary_size;
+            size_t blob_size = static_cast<uint64_t>(get_int64(s)) % ArrayBlob::max_binary_size;
```

Comment:
You should do `% (ArrayBlob::max_binary_size + 1)` to be able to also trigger the maximum blob size, which is an important edge case